### PR TITLE
feature/ads-705 Text input

### PR DESCRIPTION
**Pull request by APJohns**

### CODE REVIEW

File: src/components/InputText/InputText.tsx [hunk 1]
```diff
@@ -18,6 +18,10 @@ export interface InputTextProps extends InputHTMLAttributes<HTMLInputElement> {
   helpText?: string;
   errorText?: string;
   requiredText?: string;
+  /**
```
Comment (kaseyllabelle) [inline src/components/InputText/InputText.tsx:21]: this is only for the docs site demo, right? maybe we can note that it's for demo purposes and add a TODO to remove it and use the storybook features?

File: src/components/InputText/InputText.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+const helpText = 'This is an example of help text. It can wrap to two lines, but try not to go longer than three.';
+const errorText = 'This is an example of an error message.';
+
+export const Playground: Story = {};
+
+export const WithPlaceholder: Story = {
+  args: {
+    placeholder: 'Placeholder text'
+  }
+};
```
Comment (kaseyllabelle) [inline src/components/InputText/InputText.stories.tsx:29]: add `WithPlaceholder.storyName = 'With placeholder';` to align with casing guidelines

File: src/components/InputText/InputText.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  args: {
+    label: 'Text input'
+  }
+} satisfies Meta<typeof InputText>;
+
+export default meta;
+type Story = StoryObj<typeof InputText>;
+
+const helpText = 'This is an example of help text. It can wrap to two lines, but try not to go longer than three.';
+const errorText = 'This is an example of an error message.';
```
Comment (kaseyllabelle) [inline src/components/InputText/InputText.stories.tsx:21]: should we level these up to a shared location? we have them in fieldset too

File: src/components/InputText/InputText.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import InputText from './InputText';
+
+const meta = {
+  title: 'Components/Text input',
+  component: InputText,
+  parameters: {
+    layout: 'centered'
```
Comment (kaseyllabelle) [inline src/components/InputText/InputText.stories.tsx:9]: thoughts on padded vs centered here?

File: src/components/InputText/InputText.tsx (hunk 1 above)
Comment (APJohns) [reply src/components/InputText/InputText.tsx:21]: Could there be a use case to force validation in a product? e.g. loading a page with a form on it that already has data entered?

**Review by kaseyllabelle — APPROVED** [review]

**Pull request merged**